deploy/stage/common-values-reshare-server-left.yaml

```diff
@@ -0,0 +1,128 @@
+image: "ghcr.io/worldcoin/iris-mpc:v0.8.25"
+
+environment: stage
+replicaCount: 1
+
+strategy:
+  type: Recreate
+
+datadog:
+  enabled: true
+
+# Nginx exposes the only port required here
+ports:
+  - containerPort: 3000
+    name: health
+    protocol: TCP
+
+startupProbe:
+  httpGet:
+    path: /health
+    port: health
+
+livenessProbe:
+  httpGet:
+    path: /health
+    port: health
+
+readinessProbe:
+  periodSeconds: 30
+  failureThreshold: 10
+  httpGet:
+    path: /health
+    port: health
+
+resources:
+  limits:
+    cpu: 1
+    memory: 1Gi
+  requests:
+    cpu: 1
+    memory: 1Gi
+
+imagePullSecrets:
+  - name: github-secret
+
+nodeSelector:
+  kubernetes.io/arch: amd64
+  beta.kubernetes.io/instance-type: t3.2xlarge
+
+podSecurityContext:
+  runAsUser: 65534
+  runAsGroup: 65534
+
+serviceAccount:
+  create: true
+
+command: [ "/bin/reshare-server" ]
+
+env:
+  - name: SMPC__DATABASE__URL
+    valueFrom:
+      secretKeyRef:
+        key: DATABASE_AURORA_URL
+        name: application
+  - name: RUST_LOG
+    value: info
+  - name: ENVIRONMENT
+    value: stage
+
+service:
+  enabled: false
+
+nginxSidecar:
+  enabled: true
+  port: 8443
+  secrets:
+    enabled: true
+    volumeMount:
+      - name: mounted-secret-name
+        mountPath: /etc/nginx/cert
+    volume:
+      - name: mounted-secret-name
+        secret:
+          secretName: application
+          items:
+            - key: certificate.crt
+              path: certificate.crt
+            - key: key.pem
+              path: key.pem
+          optional: false
+  config:
+    nginx.conf: |
+      worker_processes  auto;
+      
+      error_log  /dev/stderr notice;
+      pid        /tmp/nginx.pid;
+      
+      events {
+        worker_connections  1024;
+      }
+      
+      http {
```

Comment:
@wojciechsromek feel free to take a look - I slightly reworked this from what we had in upgrade server

Reply:
Is there a need to have left/right reshare server config? The protocol does both at the same time